tests/builder_Spectrum.py

```diff
@@ -0,0 +1,36 @@
+import numpy
+from matchms import Spectrum
+
+
+class SpectrumBuilder:

```

Comment:
Things from /tests/ won't end up in the actual documentation. But it would still be nice to have a short docstring explaining what this class is meant for.